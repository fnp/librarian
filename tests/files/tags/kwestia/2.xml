<naglowek_osoba>GŁOS HESI</naglowek_osoba>

<kwestia>
<akap>
Mamuńciu, tak zimno! troszkę ciepłej wody...
</akap>
</kwestia>

<naglowek_osoba>DULSKA</naglowek_osoba>

<kwestia>
<akap>
Jeszcze czego? Hartujcie się... Felicjan! wstajesz? Wiesz? ten błazen, twój syn, nie wrócił jeszcze do domu! Co? nic nie mówisz? naturalnie.
Ojciec toleruje. Niedaleko padło jabłko od jabłoni. Ale jak będą dłużki małe --- nie zapłacę.
</akap>
</kwestia>
