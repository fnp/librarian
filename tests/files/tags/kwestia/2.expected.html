<div id="book-text">
<h4 id="i1">GŁOS HESI</h4>
<div class="kwestia">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Mamuńciu, tak zimno! troszkę ciepłej wody…</p>
</div>
<h4 id="i2">DULSKA</h4>
<div class="kwestia">
<a href="#f2" class="wl-num">2</a><p class="wl paragraph" id="f2">Jeszcze czego? Hartujcie się… Felicjan! wstajesz? Wiesz? ten błazen, twój syn, nie wrócił jeszcze do domu! Co? nic nie mówisz? naturalnie.
Ojciec toleruje. Niedaleko padło jabłko od jabłoni. Ale jak będą dłużki małe — nie zapłacę.</p>
</div>
</div>
